thirdparty/ofqf/qoscserver.cpp

```diff
@@ -136,12 +136,14 @@ void QOscServer::readyRead()
               
               		//qDebug() << " after transformation to OSC-RegExp path is" << path;
               
-              		QRegExp exp( path );
-              		foreach( PathObject* obj, paths ) {
-              		      if ( exp.exactMatch( obj->_path ) )
-              				        obj->signalData( arguments );
-              		      }
-              		}
+			QRegExp exp( path );
+                  	foreach ( PathObject* obj, paths ) {
+                        	QRegExp exp2(obj->_path);
+                        	if (exp.exactMatch(obj->_path))
+                              		obj->signalData(arguments);
```

Comment:
the above changes are nothing but formatting. Don't do that in third party code
For the below lines better adjust to the formatting used above